torch/fx/passes/utils/matcher_utils.py

```diff
@@ -99,6 +100,10 @@ def _nodes_are_equal(self, pn: Node, gn: Node) -> bool:
         if pn.op == gn.op:
             if pn.op == "placeholder" or pn.op == "output":
                 return True
+            elif pn.op == "get_attr":
+                pn_value = getattr(pn.graph.owning_module, pn.target)
+                gn_value = getattr(gn.graph.owning_module, gn.target)
+                return isinstance(pn_value, torch.Tensor) and isinstance (gn_value, torch.Tensor)
```

Comment:
I feel like we don't necessarily need to compare tensor data itself, torch.linear(10, 14) and torch.linear(12, 14) will have different weight and they should be matched, thought? 

Also regarding
> legal object
anything else I need to check here?